dev-resources/oidc/.well-known/openid-configuration

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
wait why does it need a discovery address resource? Is this like application-as-resource type thing?

Reply:
Nope, this is just for tests. I'm punning on the fact that it treats file locs + uris interchangeably

Reply:
see https://github.com/yetanalytics/lrsql/blob/e819ec40dfb68a054351d11bd5bad2d83511b8aa/src/test/lrsql/util/oidc_test.clj

Reply:
gotcha, ok cool.